c/cbindgen.toml

```diff
@@ -21,24 +21,24 @@ after_includes = """
 #ifndef FOXGLOVE_NONNULL
 #define FOXGLOVE_NONNULL
 #endif
-
-
-typedef struct foxglove_channel foxglove_channel;
 """
 
 [ptr]
 non_null_attribute = "FOXGLOVE_NONNULL"
 
 [enum]
-prefix_with_name = true
+rename_variants = "QUALIFIED_SCREAMING_SNAKE_CASE"
```

Comment:
FOXGLOVE_ERROR_VALUE_ERROR vs foxglove_error_Value_Error, seems much better to me and also fits the google style guide